CHANGELOG.md

```diff
@@ -1,11 +1,47 @@
 # SAS Viya Monitoring for Kubernetes
 
+## Version 1.1.5 (15FEB22)
+
+* **Overall**
+  * [FIX] Several spurious error messages have been suppressed
+
+* **Monitoring**
+  * [CHANGE] - The SAS Viya Welcome dashboard has been updated with a
+    cleaner design and an improved layout
+  * [EXPERIMENTAL] - Setting `ELASTICSEARCH_DATASOURCE=true` when running
+    `monitoring/bin/deploy_monitoring_cluster.sh`  will configure an
+    Elasicsearch data source for Grafana that enables viewing and querying of
+    logs from Grafana dashboards. Logging must already be deployed for this
+    to function properly.
+  * [EXPERIMENTAL] - Enhanced dashboards for SAS Go services, SAS Java
+    services, and CAS support viewing logs in Grafana. Setting
+    `ELASTICSEARCH_DATASOURCE=true` or `VIYA_LOGS_DASH=true` will deploy
+    the updated dashboards to replace the prior versions.
+
+* **Logging**
+  * [FEATURE] - Enhanced multi-line support is now enabled by default in
+    Fluent Bit
+  * [FIX] - Several status check  in scripts have been simplified to use
+    `kubectl wait`
+  * [FIX] - All logging components now specify Kubernetes resource requests
+  * [CHANGE] - Event router is now deployed to the `logging` namespace instead
+    of the `kube-system` namespace
```

Comment:
This change was done in the previous version (1.1.3).  This sprint, we fixed it to where Event Router with Work Node Placement enabled now correctly gets deployed in the logging namespace.